nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java

```diff
@@ -0,0 +1,81 @@
+package org.apache.nifi.util;
+
+import static org.junit.Assert.*;
```

Comment:
Looks good, thanks for the fix and additional tests! This line is a CheckStyle violation though, need to replace the star import with individual ones, please and thanks!

Reply:
Just fixed this, build should now pass.